eng/helix/content/runtests.cmd

```diff
@@ -59,12 +59,11 @@ if errorlevel 1 (
     REM DO NOT EXIT and DO NOT SET EXIT_CODE to 1
 )
 
-echo "Copying TestResults\* to Root\"
-for /r TestResults %x in (*.xml) do copy "%x" out
+echo "Copying TestResults\TestResults.xml to ."
+copy TestResults\TestResults.xml testResults.xml
```

Comment:
Of course, this means we're only going to get data from un-quarantined tests. We should come up with a plan for how to get this data out. Maybe we have to rethink our quarantine process.

If we ran a single run of all tests, we'd still be able to tell if they were quarantined or not by the traits. Unfortunately, the run would fail even when only quarantined tests fail.

Another option is to change how the quarantine process works and just skip tests that are quarantined. It would mean we can't monitor them for when they get non-flaky, but we'd still be able to track which tests were skipped because they were flaky and track the number of skipped flaky tests over time so that teams can drive that number down.

Food for thought at the next DoI meeting :).

Reply:
I think we should probably just add a flaky-test.yml and just run all tests in that run every day on master, and get rid of the flaky test pass in our normal PRs

Reply:
Then we'd have all the data in kusto

Reply:
Yep, I think a separate run is the solution. I think we can (and should) continue running them on PRs if we can though. The main downside is that for projects with no quarantined tests, we’ll submit a Helix work item with almost no action, it will run 0 tests. We could start with that though. I did some spelunking and it’s definitely feasible to run a little code to check a test assembly (once built) for any quarantined tests using the vstest APIs. That way we could save submitting a no-op job to helix.

Reply:
Cool maybe we can just repurpose the old helix-test pipeline to be the flaky helix test run